src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1088,7 +1097,9 @@ def create_app_service_plan(cmd, resource_group_name, name, is_linux, sku='B1',
         _linux_sku_check(sku)
     # the api is odd on parameter naming, have to live with it for now
     sku_def = SkuDescription(tier=get_sku_name(sku), name=sku, capacity=number_of_workers)
-    plan_def = AppServicePlan(location=location, tags=tags, sku=sku_def, reserved=(is_linux or None), name=name)
+    plan_def = AppServicePlan(location=location, tags=tags, sku=sku_def,
+                              reserved=(is_linux or None), is_xenon=(hyper_v or None), name=name)
+    print(plan_def)
```

Comment:
This print statement should not be here. I'm guessing this was left over from testing.